openbb_platform/CONTRIBUTING.md

```diff
@@ -538,6 +539,16 @@ As the OpenBB Platform has its own standardization framework and the data fetche
 2. Extract - `extract_data(query: ExampleQueryParams,credentials: Optional[Dict[str, str]],**kwargs: Any,) -> Dict`: makes the request to the API endpoint and returns the raw data. Given the transformed query parameters, the credentials and any other extra arguments, this method should return the raw data as a dictionary.
 3. Transform - `transform_data(query: ExampleQueryParams, data: Dict, **kwargs: Any) -> List[ExampleHistoricalData]`: transforms the raw data into the defined data model. Given the transformed query parameters (might be useful for some filtering), the raw data and any other extra arguments, this method should return the transformed data as a list of [`Data`](openbb_platform/platform/provider/openbb_core/provider/abstract/data.py) children.
 
+#### Errors
```

Comment:
This should go to the docs website.
We should consider deprecate this document and just point potential contributors to the website instead.

cc @IgorWounds @montezdesousa